src/pages/summer/Login/LoginInput/LoginInput.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+
+import './LoginInput.scss';
+
+class LoginInput extends React.Component {
+  state = {
+    idInput: '',
+    pwInput: '',
+    idVld: false,
+    pwVld: false,
+  };
+
+  gotoMain = e => {
+    const { IP_ADDRESS } = this.props;
+    e.preventDefault();
+
+    fetch(`http://${IP_ADDRESS}:8000/users/signin`, {
+      method: 'POST',
+      body: JSON.stringify({
+        email: this.state.idInput,
+        password: this.state.pwInput,
+        // name: '포켓몬 지우',
+        // phone_number: '010-0000-0000',
+      }),
+    })
+      .then(res => res.json())
+      .then(res => {
+        localStorage.setItem('TOKEN', res.access_token);
+        this.props.history.push('/summer/main');
+      });
+  };
+
+  handleInput = e => {
+    const { value, name } = e.target;
+
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  checkLoginVld = () => {
+    const { idInput, pwInput } = this.state;
+
+    const checkIdVld = /^(?=.*[@])[a-z0-9_.@]{5,}$/gi;
+    const checkedId = checkIdVld.test(idInput);
+    const checkPwVld = /^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{5,}$/;
+    const checkedPw = checkPwVld.test(pwInput);
```

Comment:
```suggestion
    const ID_VALID_REGEX = /^(?=.*[@])[a-z0-9_.@]{5,}$/gi;
    const PW_VALID_REGEX = /^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{5,}$/;
    
    const isIdValid = ID_VALID_REGEX.test(idInput);
    const isPwValid = PW_VALID_REGEX.test(pwInput);
```
- 변수명만 보고도 해당 변수에 정규식이 담겨 있고 어떤 목적인지 알 수 있도록 해주면 좋을 것 같습니다
- boolean 타입은 is- 네이밍을 사용하면 좋겠네요!